modules/mobile/app/helpers/mobile/facilities_helper.rb

```diff
@@ -1,9 +1,22 @@
 # frozen_string_literal: true
 
+require 'lighthouse/facilities/client'
+
 module Mobile
   module FacilitiesHelper
     module_function
 
+    def fetch_facilities_from_ids(user, facility_ids, include_children)
+      ids = facility_ids.join(',')
+
+      facility_ids.each do |facility_id|
```

Comment:
Not really related to your PR, but I question the value of this logging. I'll probably file a ticket to do an audit of our logging and metrics soon so this can be handled at that time.